docs/specification/README.md

```diff
@@ -294,12 +297,27 @@ A Summary Offset record contains the location of records within the summary sect
 
 A Data End record indicates the end of the data section.
 
-> Why? When reading a file from start to end, there is ambiguity when the data section ends and the summary section starts because some records (i.e. Channel Info) can repeat for summary data. The Data End record provides a clear dilineation the data section has ended.
+> Why? When reading a file from start to end, there is ambiguity when the data section ends and the summary section starts because some records (i.e. Schema, Channel Info) can repeat for summary data. The Data End record provides a clear dilineation the data section has ended.
 
 | Bytes | Name | Type | Description |
 | --- | --- | --- | --- |
 | 4 | data_section_crc | uint32 | CRC32 of all bytes in the data section. A value of 0 indicates the CRC32 is not available. |
 
+### Schema (op=0x0F)
+
+A Schema record defines an individual schema.
+
+Schema records are uniquely identified within a file by their schema ID. A Datatye record must occur at least once in the file prior to any Channel Info referring to its schema ID.
+
+| Bytes | Name | Type | Description |
+| --- | --- | --- | --- |
+| 2 | id | uint16 | A unique identifier for this schema within the file. |
+| 4 + N | encoding | String | Format for the schema. The value should be one of the [well-known schema formats](./well-known-schema-formats.md). Custom values should use the `x-` prefix. |
+| 4 + N | schema | uint32 length-prefixed Bytes | Schema should conform to the encoding. |
+| 4 + N | name | String | An identifier for the schema. The schema name should conform to any encoding requirements. |
```

Comment:
move `name` up to the top (under `id`), and rename `schema` to `data` for consistency with other messages?

Reply:
Name talks about needing to conform to the encoding - is it weird it comes before encoding?